cactusbot/handlers/command.py

```diff
@@ -89,17 +89,24 @@ def __init__(self, channel, api):
 
             if response.status == 200:
                 json = await response.json()
+                json = json["data"]["attributes"]
 
                 args = tuple(MessagePacket(
-                    *json["data"]["attributes"]["arguments"]
+                    *json["arguments"]
                 ).text.split()) + args
-                command_name = json["data"]["attributes"]["commandName"]
+                command_name = json["commandName"]
 
-                json = json["data"]["attributes"]["command"]
+                json = json["command"]
 
             else:
                 return
 
+        can_run = await self.check_role(_packet.role, json["userLevel"])
+        if not can_run:
+            return MessagePacket(
+                ("text", "Role level '{role}' or higher required.".format(
```

Comment:
This will return something like:
```
Role level '4' or higher required.
```
which will most likely mean nothing to the average user. There's a name-to-value conversion dictionary in `cactusbot/commands/command.py`... maybe move it to a class attribute of `Command`, and access `Command.ROLES` here to convert?